### ISSUE TITLE
Deploying dark nodes failure

### ISSUE THREAD

**Issue by greyhawk** [republicprotocol/darknode-cli#23]
Deploying dark nodes ...

Error: Missing required argument

  on main.tf line 55, in resource "digitalocean_droplet" "darknode":
  55:     connection {

The argument "host" is required, but no definition was found.

exit status 1

**Comment by loongy** [comment]
@greyhawk thanks for report. Could you please provide us with some more information:

- What version of the Darknode CLI are you using?
- What command did you run (please strip all API tokens)?
- What OS did you use?

**Comment by greyhawk** [comment]
macOS MOjava Version 10.14.5

```shell
Darknode CLI version 2.2.11

darknode up --name NAME --do --do-token PERSONAL-ACCESS-TOKEN
```

```shell
Initializing the backend...

Initializing provider plugins...
- Checking for available provider plugins...
- Downloading plugin for provider "digitalocean" (terraform-providers/digitalocean) 1.4.0...

The following providers do not have any version constraints in configuration,
so the latest version was installed.

To prevent automatic upgrades to new major versions that may contain breaking
changes, it is recommended to add version = "..." constraints to the
corresponding provider blocks in configuration, with the constraint strings
suggested below.

* provider.digitalocean: version = "~> 1.4"

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.

Deploying dark nodes ...

Error: Missing required argument

  on main.tf line 55, in resource "digitalocean_droplet" "darknode":
  55:     connection {

The argument "host" is required, but no definition was found.

exit status 1
```

**Comment by tok-kkk** [comment]
@greyhawk  Could you please also provide your `terraform` version?

**Comment by greyhawk** [comment]
@tok-kkk 
```shell
terraform -v
Terraform v0.12.3
```

**Comment by tok-kkk** [comment]
Hi @greyhawk. Terraform's latest version is 0.12.X which is a major update and is not backwards compatible([updating to v0.12](https://www.terraform.io/upgrade-guides/0-12.html)). A quick fix is to downgrade your terraform to 0.11.x and run the `darknode up` command. We'll also add support for the new version of `terraform` in the near future.

**Comment by greyhawk** [comment]
thanks!
